mcs/class/System.Drawing/System.Drawing/gdipFunctions.cs

```diff
@@ -47,6 +47,15 @@ namespace System.Drawing
 	/// GDI+ API Functions
 	/// </summary>
 	internal static class GDIPlus {
+
+#if NETSTANDARD1_6
+		public const CharSet GdiPlusCharSet = CharSet.Unicode;
+		public const UnmanagedType GdiPlusCustomMarshaler = UnmanagedType.Interface;
```

Comment:
The custom marshaler is always `ComIStreamMarshaler`, and [there's a comment at the definition of `ComIStreamMarshaler`](https://github.com/mono/mono/blob/master/mcs/class/System.Drawing/System.Drawing/ComIStreamMarshaler.cs#L46) that basically reads:

> This custom marshaler should be replaced with UnmanagedType.Interface
> Provides identical behaviour under Mono and .NET Framework

That comment is 12 years old, though :)